packages/jest-worker/src/workers/processChild.ts

```diff
@@ -40,6 +43,10 @@ const messageListener: NodeJS.MessageListener = request => {
       const init: ChildMessageInitialize = request;
       file = init[2];
       setupArgs = request[3];
+      heartbeatIntervalValue = request[4];
+      monitorHeartbeat = setInterval(() => {
+        sendParentMessageHeartbeat();
+      }, heartbeatIntervalValue);
```

Comment:
```suggestion
      }, heartbeatIntervalValue).unref();
```

here as well, I think?